IoTCoreDefaultApp/IoTCoreDefaultApp/IoTCoreDefaultApp.csproj

```diff
@@ -11,8 +11,8 @@
     <AssemblyName>IoTCoreDefaultApp</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.10240.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.10240.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion>10.0.10545.0</TargetPlatformVersion>
```

Comment:
It won't compile if you do that iirc

Sent from my Windows Phone
________________________________
From: Brian Fjeldstad<mailto:notifications@github.com>
Sent: ‎10/‎26/‎2015 4:01 PM
To: ms-iot/samples<mailto:samples@noreply.github.com>
Cc: Kara Ottewell<mailto:karaottewell@outlook.com>
Subject: Re: [samples] Changes to add the Bluetooth tab to Settings so that BT Inbound and Outbound Pairing can be done (#130)

> @@ -11,8 +11,8 @@
>      <AssemblyName>IoTCoreDefaultApp</AssemblyName>
>      <DefaultLanguage>en-US</DefaultLanguage>
>      <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
> -    <TargetPlatformVersion>10.0.10240.0</TargetPlatformVersion>
> -    <TargetPlatformMinVersion>10.0.10240.0</TargetPlatformMinVersion>
> +    <TargetPlatformVersion>10.0.10545.0</TargetPlatformVersion>

leave this as 10240

---
Reply to this email directly or view it on GitHub:
https://github.com/ms-iot/samples/pull/130/files#r43065615